### PR TITLE
[openrr-apps] Supprot Navigation / MoveBase in robot_command

### PR DESCRIPTION
**Pull request by mitsuharu-kojima**

### CODE REVIEW

File: openrr-apps/src/robot_command.rs [hunk 1]
```diff
@@ -98,6 +101,28 @@ pub enum RobotSubCommand {
     Speak { message: Vec<String> },
     /// Execute an external command.
     ExecuteCommand { command: Vec<String> },
+    /// Get navigation current pose.
+    GetNavigationCurrentPose {},
```
Comment (OTL) [inline openrr-apps/src/robot_command.rs:105]: Is it possible to `GetNavigationCurrentPose {}` -> `GetNavigationCurrentPose`?

It is ok to do it in another change for all members.

**Review by OTL — COMMENTED** [review]
Please fix commit message typo at frist

File: openrr-apps/src/robot_client.rs [hunk 2]
```diff
@@ -95,6 +97,28 @@ impl RobotClient {
...
+                Some(Arc::new(RosNavClient::new_from_config(
+                    ros_navigation_client_config.clone(),
+                )) as Arc<dyn Navigation>)
+            } else if config.use_navigation_urdf_viz_web_client {
+                let urdf_viz_client = Arc::new(UrdfVizWebClient::default());
+                urdf_viz_client.run_thread();
+                Some(urdf_viz_client as Arc<dyn Navigation>)
+            } else {
+                None
+            };
```
Comment (OTL) [inline openrr-apps/src/robot_client.rs:120]: I want to refactor these like below

```rust
let navigation = create_navigation(xxx);
```

```rust
fn create_navigation(xxx) {
  #[cfg(feature = "ros")]
  create_navigation_with_ros();
  #[cfg(not(feature = "ros"))]
  create_navigation_without_ros();
}
```

But we have similar structure already, let's do it another change....

**Review by OTL — APPROVED** [review]

File: openrr-apps/src/robot_client.rs (hunk 2 above)
Comment (mitsuharu-kojima) [reply openrr-apps/src/robot_client.rs:120]: I will move these feature(ros) dependent code out of robot client.

**Pull request merged**